usagov/vote-gov-drupal
# Feature/vote 2021/datalayer

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2021

## Description
Add a dataLayer array to the head of the html files. Build it conditionally based on the requirements specified in the ticket.

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. On the following pages, view the source in the browser and confirm the data in the dataLayer array is what we expect
http://vote-gov.lndo.site/
http://vote-gov.lndo.site/es
http://vote-gov.lndo.site/register
http://vote-gov.lndo.site/register/alabama
http://vote-gov.lndo.site/guide-voting
http://vote-gov.lndo.site/voting-new-us-citizen
http://vote-gov.lndo.site/about-us
http://vote-gov.lndo.site/about-us
http://vote-gov.lndo.site/accessibility

## Checklist for the Developer




## Checklist for the Peer Reviewers
